0004-post-tags.md

```diff
@@ -0,0 +1,299 @@
+- Feature Name: Post Tags
+- Start Date: 2023-11-04
+- RFC PR: [LemmyNet/rfcs#0004](https://github.com/LemmyNet/rfcs/pull/4)
+- Lemmy Issue: [LemmyNet/lemmy#317](https://github.com/LemmyNet/lemmy/issues/317)
```

Comment:
Imo we can change the rfc numbering to start from 1 and simply increment, instead of using the pull request number. Otherwise we will have lots of numbers skipped because of issues which will look ugly.

Reply:
It's what the Rust people also do in their repo (and it's in our readme as well). Their numbers also have holes but the advantage is they fit with the github "hashtag" linking and avoid confusion. Since the rust people tend to know what they're doing I'd suggest to keep the same method

Reply:
Yep its true, using manual numbers means that RFC numbers would have to be changed if there are multiple RFCs open and merged at overlapping times or not merged at all. So using the PR number is much easier.

Reply:
Agree with keeping the numbering as is, it makes it very clear where which RFC was merged. Plus as mentioned any other numbering would mess up the workflow a bit.

I'll just close this as resolved then.